### DISCUSSION TITLE
RT window and Median.RT.Prediction.Acc

### DISCUSSION THREAD

**Discussion by haianhle** [vdemichev/DiaNN#1293] [Q&A]
Hi Vadim,

Usually in the log file there is always "RT window set to XYZ", and in the stats summary there is a "Median.RT.Prediction.Acc". Are these numbers related? 

I have series of run where suddenly DIANN stopped working after a certain injection. Looking at TIC for both MS1, MS2, XICs for a number of peaks to check retention time, everything seems fine. I was running samples on a timsTOF HT and the TIMS IN pressure was not changing, temperature was stable, chiller was full, LC pressure was not unusual. I'm at a loss how that suddenly happened. 

When I plot Median Mass Acc MS1 and MS2 from stats it does seem that there was a trend. 
![image](https://github.com/user-attachments/assets/c8976999-0ba6-48e9-98a6-7c86caac4e4b)

Looking at the Median.RT.Prediction.Acc the runs that DIANN failed to process mass calibration got this jump up significantly. 
![image](https://github.com/user-attachments/assets/42ec6acf-7dd1-44c6-a7c1-4cc7b3706795)

If you have any advice or ideas I would truly appreciate it. Thank you!

**Comment by vdemichev** [comment]
Hi,

Yes, they are related. All things equal, wider RT window will lead to slightly higher median error. But that's not the source of what you see.

>  I'm at a loss how that suddenly happened.

Please try DIA-NN 1.9.2 with --mass-acc-cal 100 and some public spectral library.

Best,
Vadim

**Comment by haianhle** [comment]
Hi Vadim, these samples were search with v1.8.1 with a library. I will try to re-search with --mass-acc-cal 100 and v1.9.2 as you recommended. Thank you. Previously when I changed Mass accuracy and MS1 accuracy settings to 40 with  --mass-acc 40 --mass-acc-ms1 40, I was able to process the rest of the samples but the IDs suffered significantly. 
![image](https://github.com/user-attachments/assets/d3c6cbe6-1710-4399-99a8-1a39ae3ef7fc)

![image](https://github.com/user-attachments/assets/69fe3ece-0ff6-453c-a3cd-551954ac68f2)

MS intensities look consistent. The samples with low intensities are no sample controls.
![image](https://github.com/user-attachments/assets/29b0acec-52f9-4c9f-8ea9-eecf3d127bfc)

**Comment by haianhle** [comment]
Hi @vdemichev  thank you so much for your suggestion. It appears that with DIANN 1.9.2 mass-acc-cal 100 the identifications appear a lot more consistent versus v1.8.1 with mass-acc 40. Could you please help explain how 1.9.2 was able to sort of rescue the runs with high mass errors? Thank you again :)

v1.8.1 
![image](https://github.com/user-attachments/assets/b936de80-b83d-4f8f-a151-3aa3dd71a848)


v1.9.2
![image](https://github.com/user-attachments/assets/d94d8ddd-3c63-4e4a-ae60-756f51832fff)

**Reply by vdemichev** [reply]
> Could you please help explain how 1.9.2 was able to sort of rescue the runs with high mass errors? 

Resolution is fine but mass calibration appears to be off by a lot in those runs, with --mass-acc-cal 100 DIA-NN 1.9.2 can calibrate for up to 100ppm.
